rules.mk

```diff
@@ -87,21 +104,17 @@ push-hook-bootkit push-hook-docker:
 	docker buildx build --platform $$platforms --push -t $(ORG)/$(container):$T $(container)
 
 .PHONY: dist
-dist: out/$T/rel/hook-$T.tar.gz ## Build tarball for distribution
-out/$T/rel/hook-$T.tar.gz: out/$T/rel/hook.tar
-	pigz < $< > $@
-
-out/$T/rel/hook.tar:
-	rm -rf out/$T/rel/dist
-	mkdir -p out/$T/rel/dist
-	for a in $(arches); do
-		cp out/$T/rel/$$a/initrd.img out/$T/rel/dist/initramfs-$$a
-		cp out/$T/rel/$$a/kernel out/$T/rel/dist/vmlinuz-$$a
-	done
-	cd out/$T/rel/dist && tar -cvf ../$(@F) ./*
+dist: ## Build tarballs for distribution
+$(dist-files):
+	cp $< $@
 
+.PHONY: dbg-dist
+dbg-dist: out/$T/dbg/hook_$(DOCKER_2_HW_$(ARCH)).tar.gz ## Build debug enabled tarball
+
+.PHONY: deploy
 deploy: dist ## Push tarball to S3
-ifeq ($(shell git rev-parse --abbrev-ref HEAD),main)
-	s3cmd sync ./out/hook-$T.tar.gz s3://s.gianarb.it/hook/$T.tar.gz
-	s3cmd cp s3://s.gianarb.it/hook/hook-$T.tar.gz s3://s.gianarb.it/hook/hook-main.tar.gz
-endif
+	exit 1
+	for f in out/$T/rel/hook_*.tar.gz; do
+	    s3cmd sync $$f s3://s.gianarb.it/hook/$T/
+	    s3cmd cp s3://s.gianarb.it/hook/$T/$$(basename $$f) s3://s.gianarb.it/hook/latest/
+	done
```

Comment:
This is not used any more. I would vote to remove it all together over modifying it in any way.